.github/workflows/Compile.yml

```diff
@@ -23,4 +23,4 @@ jobs:
             ~/.bitcoin-s/binaries
           key: ${{ runner.os }}-cache
       - name: Compile and Check Formatting
-        run: sbt ++2.12.12 test:compile scalafmtCheckAll
+        run: sbt +test:compile scalafmtCheckAll
```

Comment:
I think it's best to enforce 2.12 for this since most of us are using 2.13 locally

Reply:
The `+` compiles for both versions